themes/gruvbox-dark.json

```diff
@@ -0,0 +1,937 @@
+{
```

Comment:
Thanks for re-organising this.
Since you're doing this, would you mind running a `Format Document` command on this file, so that it's formatted properly?

Reply:
ok I'll do this